src/main/java/racing/controller/Controller.java

```diff
@@ -13,16 +13,14 @@ public class Controller {
 
     private final InputView inputView;
     private final OutputView outputView;
-    private final Parser parser;
 
-    public Controller(InputView inputView, OutputView outputView, Parser parser) {
+    public Controller(InputView inputView, OutputView outputView) {
         this.inputView = inputView;
         this.outputView = outputView;
-        this.parser = parser;
     }
 
     public void startGame() {
-        List<String> names = parser.parse(inputView.readNames());
+        List<String> names = inputView.readNames();
         Cars cars = new Cars(names.stream().map(Car::new).collect(Collectors.toList()));
```

Comment:
```suggestion
        Cars cars = new Cars(names.stream().map(Car::new).toList();
```
저번 스텝에서부터 있던 로직이긴한데 `collect(Collectors.toList())` 는 `toList()` 로 바꿔서 사용할 수도 있어요!
그런데 두 방식의 결과물엔 약간의 차이가 있는데요, 이번 기회에 한 번 학습해보는 것도 좋을 것 같네요!

Reply:
`Stream.toList()`와 다르게 `Collectors.toList()`는 리턴되는 List가 수정이 가능해요

제가 이전 코드에서는 아래와 같이 코드를 작성하여 toList를 사용하면 `Collections.sort(cars);`에서 에러가 발생했어요! 

```
public Cars(List<Car> cars) {
    validate(cars);
    this.cars = cars;
}
```

```
public List<Car> findWinners() {
    Collections.sort(cars);
    return cars.stream().filter(car -> car.isSame(cars.get(0))).toList();
}
```

하지만 아래 피케이의 피드백에 따라 불변 객체로 만들기 위해 생성자를 수정하면서 `toList()`를 사용해도 에러가 발생하지 않네요!